lib/dialects/mssql/query.js

```diff
@@ -164,8 +164,8 @@ module.exports = (function() {
   Query.prototype.handleShowTablesQuery = function(results) {
     return results.map(function(resultSet) {
       return {
-        tableName: resultSet.TABLE_NAME,
-        schema: resultSet.TABLE_SCHEMA
+        tableName: resultSet.tableName,
+        schema: resultSet.schema
```

Comment:
@Groxot this shouldn't be a problem since the dialect handles the translation. Are you experiencing some sort of error? Are you maybe directly calling one of these methods or something? Can you please post some sort of code snippet to describe the problem you're facing, maybe it's easier in this case to "speak through code"